js/ui/scheduler/appointmentPopup/popup.js

```diff
@@ -25,6 +25,8 @@ const POPUP_CONFIG = {
     maxHeight: '100%',
     showCloseButton: false,
     showTitle: false,
+    preventScrollEvents: false,
```

Comment:
Don't we wanna add a QUnit test to fix the scenario, save a ticket ID and so on?